libcxx/include/format

```diff
@@ -237,21 +237,21 @@ namespace std {
 #  include <cstdint>
 #  include <cstdlib>
 #  include <cstring>
-#  include <cwchar>
 #  include <initializer_list>
 #  include <limits>
+#  include <locale>
 #  include <new>
 #  include <optional>
+#  include <queue>
+#  include <stack>
 #  include <stdexcept>
 #  include <string>
 #  include <string_view>
 #  include <tuple>
-#endif
 
-#if !defined(_LIBCPP_REMOVE_TRANSITIVE_INCLUDES) && _LIBCPP_STD_VER <= 20
```

Comment:
Why are we changing the behaviour here? This PR is protecting the include for `cwchar`, I think this needs to be done in a separate commit/PR if this is not related. If it is, can there be an explanation as to why?

Reply:
We're not changing the behaviour. If you expand this window to see the lines above, you'll see that the exact same conditional guards the includes above.

I merged the two sections to make the code clearer.

Reply:
Can that be made clearer in the commit message?

Reply:
As I think we'll loose or bury this convo if I force-push, I've updated the description in my automated first comment in this thread. I'll update the description when I merge it. Unless there's a better way to handle this.

Reply:
I have no idea whether that will do what you want, so good luck with getting the right version of the commit message into the final commit!

Reply:
If you force-push over the current commit it should keep this conversation, not sure GitHub will delete it.

Reply:
ok, let me give it a shot now that I've got approval.

Reply:
Ah that did work. Googling says comments don't disappear if the lines they're on don't change. Seems like they do still disappear if the lines change: https://github.com/orgs/community/discussions/3478.